src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -630,16 +631,22 @@ def cancel_job(self, job_id: str, parent_job_id: str = None) -> None:
                 self._running_jobs[job_id]["refresh"] = is_refreshing
                 del self._running_jobs[job_id]["canceling"]
 
-    def retry_job(self, job_id: str):
-        if job_id is None:
-            raise ValueError("Job id required for retrying")
-        if job_id not in self._running_jobs:
-            raise ValueError(f"No job present with id {job_id}")
+    def retry_jobs(self, job_id_list: List[str]):
+        for job_id in job_id_list:
+            if job_id is None:
+                raise ValueError("Job id required for retrying")
+            if job_id not in self._running_jobs:
+                raise ValueError(f"No job present with id {job_id}")
```

Comment:
The downstream `jobmanager.py` part that does `job_id` verification and throws errors about individual `job_id`s up to `jobcomm.py`

Reply:
This is an annoying pattern (not your fault, obviously) -- it means that if you have a whole set of errors, you have to eliminate them one by one. A much better design is to collect all the errors and send them in a single response. I would therefore rewrite this code to collect a list of job_ids not in `self._running_jobs`; if that list has length > 0, raise a value error (the text can be something like "No jobs present with the following IDs: ")